packages/execute-driver/test/unit/plugin-specs.js

```diff
@@ -0,0 +1,12 @@
+import { ExecuteDriverPlugin } from '../../index';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.use(chaiAsPromised);
+const should = chai.should();
+
+describe('execute driver plugin', function () {
+  it('should exist', function () {
+    should.exist(ExecuteDriverPlugin);
```

Comment:
imo this isn't valuable, but given there are no other unit tests, maybe it is, relatively speaking :smile:

Reply:
just wanted to make sure there was a single test :-) this plugin is primarily tested via e2e